### PR TITLE
remove cloudsepcific attrs

### PR DESCRIPTION
**Pull request by ggramal**

### CODE REVIEW

File: charts/postgres/Chart.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ dependencies:
     condition: postgres-main.enabled
   #https://github.com/zalando/postgres-operator/tree/v1.10.1/charts/postgres-operator
```
Comment (devops-dmitry) [inline charts/postgres/Chart.yaml:10]: 1.14.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+  * `environments`:
+    * `defaultValuesTpl` adjusted in order to reflect `environment` chart changes
+
+New features:
+  * `argocd,cert-manager,thanos,grafana,prometheus,prometheus`:
+    * new global attribute - registry
+  * `environment`:
+    * new attribute - registry
+  * `postgres`:
+    * update postgre-operator subchart 1.10.1 -> 1.14.0
```
Comment (devops-dmitry) [inline CHANGELOG.md:21]: postgres-operator

File: charts/postgres/Chart.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ dependencies:
...
   - name: postgres-operator
-    version: v1.10.1
+    version: v1.14.0
```
Comment (devops-dmitry) [inline charts/postgres/Chart.yaml:12]: I think we need to update the crds

File: CHANGELOG.md (hunk 2 above)
Comment (ggramal) [reply CHANGELOG.md:21]: hmm probably you are right :thinking: . But updating postgres operator is a totaly different task. Maybe i should revert my changes wdyt?

Comment (devops-dmitry) [reply CHANGELOG.md:21]: Yes, let's leave it for a full-fledged operator update

Comment (ggramal) [reply CHANGELOG.md]: ok reverted

**Pull request merged**